dts/bindings/adc/io-channel-nexus.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for io-channel nexus nodes
+
+properties:
+  io-channel-map:
+    type: compound
+    required: true
+
+  io-channel-map-mask:
+    type: array
+
+  io-channel-map-pass-thru:
+    type: array
+
+  "#io-channel-cells":
+    type: int
+    required: true
+    description: Number of items to expect in a ADC specifier
```

Comment:
I think, strictly speaking, that these are derived from the Linux iio bindings. The iio bindings are useful for ADCs but are not limited to ADCs. So perhaps this should be moved to dts/bindings/iio, and this "ADC specifier" portion should be changed to "channel specifier, e.g. ADC channel" or something similar?

Reply:
That's correct. We've taken care of that.